### PR TITLE
doc: Review mapping ORM and ODM cookbook

### PR DESCRIPTION
**Pull request by GromNaN**
|      Q       |   A
|------------- | -----------
| Type         | doc
| BC Break     | no
| Fixed issues | -

#### Summary

- Adding property types
- Add use statement with namespace alias (`ORM` and `ODM`) so that attribute namespaces are more legible.
- Maybe we should have a full example with both attribute sets in the same class.

### CODE REVIEW

File: docs/en/cookbook/mapping-classes-to-orm-and-odm.rst [hunk 1]
```diff
@@ -21,18 +21,16 @@ for multiple Doctrine persistence layers:
 
     class BlogPost
     {
-        private $id;
-        private $title;
-        private $body;
-
-        // ...
```
Comment (GromNaN) [inline docs/en/cookbook/mapping-classes-to-orm-and-odm.rst:28]: With public properties, we don't need anything more. This comment is a complete waste of space.

File: docs/en/cookbook/mapping-classes-to-orm-and-odm.rst [hunk 2]
```diff
@@ -48,21 +46,20 @@ First define the mapping for the ORM:
...
-
-            #[Column(type: 'string')]
-            private $title;
+            #[ORM\Id]
+            #[ORM\Column(type: 'int')]
+            pubic string $id;
 
-            #[Column(type: 'text')]
-            private $body;
+            #[ORM\Column(type: 'string')]
```
Comment (GromNaN) [inline docs/en/cookbook/mapping-classes-to-orm-and-odm.rst:59]: Oups?

**Review by alcaeus — APPROVED** [review]

**Comment by GromNaN** [comment]
@alcaeus added tests and updated the example to ensure we have the same behavior for ORM and ODM.

File: composer.json [hunk 3]
```diff
@@ -40,6 +40,7 @@
         "ext-bcmath": "*",
         "doctrine/annotations": "^1.12 || ^2.0",
         "doctrine/coding-standard": "^12.0",
+        "doctrine/orm": "^3.2",
```
Comment (alcaeus) [inline composer.json:43]: Oof 😅

File: tests/Documentation/MappingOrmAndOdm/MappingOrmAndOdmTest.php [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+class MappingOrmAndOdmTest extends BaseTestCase
+{
+    public function testTest(): void
+    {
+        // Init ORM
+        $config     = ORMSetup::createAttributeMetadataConfiguration(
+            paths: [__DIR__],
+            isDevMode: true,
+        );
+        $connection = DriverManager::getConnection([
```
Comment (alcaeus) [inline tests/Documentation/MappingOrmAndOdm/MappingOrmAndOdmTest.php:23]: Should we add checks and skip this test if `pdo_sqlite` is not installed?

Comment (GromNaN) [reply tests/Documentation/MappingOrmAndOdm/MappingOrmAndOdmTest.php:23]: Done.

**Pull request merged**